### DISCUSSION TITLE
Release of latest realm-swift doesn't include latest realm-core

### DISCUSSION THREAD

**Discussion by mwoollard** [realm/realm-cocoa#7680] [General]
Wondering when the latest realm-core with fix for watch-os builds I need will be incorporated into realm-swift? See both were updated last week but realm-swift is still using previous core. Thanks

**Comment by tgoyne** [comment]
The most recent core release introduced bugs which were caught by the Swift test suite and need to be fixed before we can pull it in.

**Reply by mwoollard** [reply]
OK thanks
